clippy_lints/src/doc/mod.rs

```diff
@@ -420,6 +421,38 @@ declare_clippy_lint! {
     "require every line of a paragraph to be indented and marked"
 }
 
+declare_clippy_lint! {
+    /// ### What it does
+    /// Checks if the first line in the documentation of items listed in module page is not
```

Comment:
```suggestion
    /// Checks if the first line in the documentation of items listed in module page is
```

Clippy lints usually say something like "Checks for [BAD PATTERN]", so in this case your lint might say that it checks whether the first line _is_ too long [which would be bad].

Reply:
Good point!